crates/revm/revm-inspectors/src/tracing/js/mod.rs

```diff
@@ -101,7 +103,7 @@ impl JsInspector {
             .cloned()
             .ok_or(JsInspectorError::ResultFunctionMissing)?;
         if !result_fn.is_callable() {
-            return Err(JsInspectorError::ResultFunctionMissing)
+            return Err(JsInspectorError::ResultFunctionMissing);
```

Comment:
please no more trailing semicolons 😭

Reply:
I'm sorry - there's a bug in `cargo +nightly fmt --all` it seems on darwin where it's not respecting the `rustfmt.toml`, gotta go in and do it manually 🫠